test/test_requirements.py

```diff
@@ -5,8 +5,14 @@
 import json
 import re
 import subprocess
+import sys
 import tempfile
-from importlib.metadata import version
+
+if sys.version_info >= (3, 8):
+    from importlib.metadata import version
+else:
+    from importlib_metadata import version
```

Comment:
I'm confused: are lines 12 and 14 supposed to be the same?

Reply:
They are not the same! `importlib.metadata` is in Python standard library and `importlib_metadata` is a backport for older Pythons :)
Note the different character in the middle, `.` vs `_`.

Reply:
Ahah, confusing but understandable.  Thanks!